transit_odp/timetables/tasks.py

```diff
@@ -627,61 +629,81 @@ def task_rerun_timetables_etl_specific_datasets():
     provided in a csv file available in AWS S3 bucket
     """
     csv_file_name = "rerun_timetables_etl.csv"
-    dataset_ids = read_datasets_file_from_s3(csv_file_name)
-    if not dataset_ids:
-        logger.info("No valid dataset IDs found in the file.")
+    dataset_ids, dataset_revision_ids, type = read_datasets_file_from_s3(csv_file_name)
+
+    if not dataset_ids and not dataset_revision_ids:
+        logger.info("No valid dataset IDs or dataset revision IDs found in the file.")
         return
-    logger.info(f"Total number of datasets to be processed: {len(dataset_ids)}")
-    timetables_datasets = Dataset.objects.filter(id__in=dataset_ids).get_active()
+
+    timetables_datasets = []
```

Comment:
if not type:
   logger.info("Not a valid type/column of dataset IDs or dataset revision IDs found in the file.")
   return